### ISSUE TITLE
Can't install Atom

### ISSUE THREAD

**Issue by Quantum64** [atom/atom#5671]
Atom became horribly broken to the point where I couldn't open it, so I uninstalled it from the Add/Remove programs menu, and attempted to reinstall it, bit I get this error: https://gist.github.com/Quantum64/ac67c7fd4d8cec4a08bc

Running OS Windows 10 Build 9926

I have also tired deleting the atom folder in AppData.

**Comment by 50Wliu** [comment]
Errors in the Gist file:
`CheckForUpdateImpl: Failed to load local releases, starting from scratch: System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\Users\Dylan\AppData\Local\atom\packages\RELEASES'.`
followed by `CheckForUpdateImpl: First run or local directory is corrupt, starting from scratch`
followed by `IEnableLogger: Failed to copy Update.exe to C:\Users\Dylan\AppData\Local\atom\Update.exe: System.IO.IOException: The process cannot access the file 'C:\Users\Dylan\AppData\Local\atom\Update.exe' because it is being used by another process.` (Second install attempt?)
followed by `IEnableLogger: Failed to update local releases file: System.IO.IOException: The file exists.` (Third install attempt?)
followed by `CheckForUpdateImpl: Failed to load local releases, starting from scratch: System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\Users\Dylan\AppData\Local\atom\packages\RELEASES'.` (Fourth install attempt?)

/cc @paulcbetts

**Comment by paulcbetts** [comment]
@Quantum64 Hey Dylan, the easiest thing to do is to just delete all of `C:\Users\Dylan\AppData\Local\atom` via Explorer, then try again. If files are locked, reboot then try that.

**Comment by Quantum64** [comment]
@paulcbetts I deleted the Local/atom folder, restarted my computer (for good measure), ran the installer again, and got the same error. https://gist.github.com/Quantum64/45c265291a5c401fe883

**Comment by paulcbetts** [comment]
@Quantum64 That's super bizarre, it's basically saying we can't create a temp file. Google says that your `C:\Windows\Temp` folder might have bajillions of files in it.

**Comment by Quantum64** [comment]
@paulcbetts The Temp folder looks fine.  It has 57 items in total.  I did have to give myself "access" to it in the security properties to view the files.

I also tried redownloading the installer from the website, as well as an older installer that I had from about a month ago.  All fail with the same result.

**Comment by 50Wliu** [comment]
They're temp files, so it shouldn't matter that much if you delete them (or if you don't feel safe doing that run the Disk Cleanup tool that's included with Windows).

**Comment by Quantum64** [comment]
Ok then, It's deleting 4 million files and it's going to take 26 min.  Most of the files have a size of 0 bytes. :confused:

**Comment by 50Wliu** [comment]
Was the 19th the day you tried to install Atom?  (Asking this because I saw an issue on the Squirrel repo that might be related to this)

**Comment by Quantum64** [comment]
No.  The only thing I installed on the 19th was an update to GitHub for Windows, but I doubt that's what caused it.

I think this overload of temporary files caused other issues with Atom as well.  For instance, I could not open any files at all. The program would launch, but simply refuse to open files.  Perhaps it needs to create a temporary file to do so?  That's why I did a reinstall in the first place.
